service/frontend/accessControlledHandler.go

```diff
@@ -326,6 +337,7 @@ func (a *AccessControlledWorkflowHandler) PollForActivityTask(
 		APIName:    "PollForActivityTask",
 		DomainName: request.GetDomain(),
 		TaskList:   request.TaskList,
+		Permission: authorization.PermissionRead,
```

Comment:
This is PermissionWrite. 
PollForActivity will get the activity task and also start the activity, which trigger to write the event "ActivityTaskStarted" event in the history.